src/NFCManager.cpp

```
@@ -1759,16 +1759,74 @@ bool NFCManager::executeTigerTagWrite(const NFCWriteRequest& request) {
     }
-    return ok;
+    if (inRun) runCount++;
+
+    if (runCount == 0) {
+        Serial.println("NFCManager: WRITE_TIGERTAG no changes, skipping write");
+        LogBuffer::getInstance().logPrintf("Write TigerTag: no changes\n");
+        return true;
+    }
+
+    uint8_t totalPages = 0;
+    for (uint8_t r = 0; r < runCount; ++r) {
```

Clang (14.0.6)

[warning] loop variable name 'r' is too short, expected at least 2 characters

(readability-identifier-length)